indico/modules/events/management/controllers/actions.py

```diff
@@ -82,11 +82,35 @@ class RHMoveEvent(RHManageEventBase):
     def _process_args(self):
         RHManageEventBase._process_args(self)
         self.target_category = Category.get_or_404(int(request.form['target_category_id']), is_deleted=False)
-        if not self.target_category.can_create_events(session.user):
-            raise Forbidden(_('You may only move events to categories where you are allowed to create events.'))
+
+    def _check_access(self):
+        if not self.target_category.can_propose_events(session.user):
+            raise Forbidden(_('You may only move events to categories where you are allowed to propose events.'))
```

Comment:
That error sounds extremely confusing...

Reply:
How about?
`You may only move events to categories you are allowed to.`
Might seem confusing to make a distinction between allowed to create/allowed to request moves.

Reply:
"You may not move events to this category"? I guess you usually do not get there anyway since it's disabled in the UI...